src/jwt.ts

```diff
@@ -126,7 +129,8 @@ export class Verifier<T extends BaseClaims> {
 
     const joseKey = await jose.JWK.asKey(myPublicKey);
 
-    const verifiedToken = await jose.JWS.createVerify(joseKey).verify(token);
+    const verifier = jose.JWS.createVerify([joseKey]);
+    const verifiedToken = await verifier.verify(token);
```

Comment:
Why change it to two steps? For debugging or readability?

Reply:
For readability